README.md

```diff
@@ -243,27 +215,29 @@ Gateway settings:
 
   1. **ValueId Topics**: _Automatically configured_. The topic where zwave values are published will be:
 
-     `<mqtt_prefix>/<?node_location>/<node_id>/<class_id>/<instance>/<index>`
+     `<mqtt_prefix>/<?node_location>/<nodeId>/<commandClass>/<endpoint>/<property>/<propertyKey>`
 
      - `mqtt_prefix`: the prefix set in Mqtt Settings
      - `node_location`: location of the Zwave Node (optional, if not present will not be added to the topic)
-     - `node_id`: the unique numerical id of the node in Zwave network
-     - `class_id`: the numerical class id of the value
-     - `instance`: the numerical value of value instance
-     - `index`: the numerical index of the value
+     - `nodeId`: the unique numerical id of the node in Zwave network
+     - `commandClass`: the command class number of the value
+     - `endpoint`: the endpoint number (if the node has more then one endpoint)
+     - `property`: the value [property](https://zwave-js.github.io/node-zwave-js/#/api/valueid)
+     - `propertyKey`: the value [propertyKey](https://zwave-js.github.io/node-zwave-js/#/api/valueid)
 
-  2. **Named Topics**: _Automatically configured_. **DEPRECATED** After a discussion with Openzwave author lib we discourage users to use this configuration as we cannot ensure that value labels will be the same, they could change in future versions (and also they depends on localization added in OZW 1.6). You can find more info [HERE](https://github.com/OpenZWave/Zwave2Mqtt/issues/22)
+  2. **Named Topics**: _Automatically configured_. **DEPRECATED** After a discussion with zwave-js author lib we discourage users to use this configuration as we cannot ensure that value labels will be the same, they could change in future versions (and also they depends on localization added in OZW 1.6). You can find more info [HERE](https://github.com/zwave-js/zwavejs2mqtt/issues/22)
```

Comment:
Do we even need to support this in the new world can we just strip it out?

Reply:
Actually this is not deprecated anymore as there is no localization in zwavejs (in reality neither ozw had a real localization) but now strings are fixed so this can be used safely

Reply:
I think that just replacing `openzwave` with `zwave-js` here is not correct :)